quantumapp/server.py

```
@@ -0,0 +1,248 @@
+        with redirect_stdout(buf_out), redirect_stderr(buf_err):
+            fn(*args, **kwargs)
+        out = buf_out.getvalue()
+        err = buf_err.getvalue()
+        msg = out + ("\n[stderr]\n" + err if err else "")
+        return True, msg.strip()
+    except SystemExit:
+        out = buf_out.getvalue()
+        err = buf_err.getvalue()
+        msg = out + ("\n[stderr]\n" + err if err else "")
+        return True, msg.strip()
+    except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)